cypress/e2e/pages/governance/create/index.spec.ts

```diff
@@ -0,0 +1,593 @@
+        );
+      });
+    });
+
+    it("should be able to edit proposal", () => {
+      cy.findByTestId("Governance.Create.Step1.Edit")
+        .should("be.visible")
+        .click();
+
+      cy.findByTestId("Governance.Create.Step1.TextArea.NameOfProposal").within(
+        () => {
+          cy.get("textarea").click().clear().type("Edited Proposal");
+        }
+      );
+      cy.findByTestId("Governance.Create.Step1.ReviewProposal").click();
+      cy.findByTestId(
+        "Governance.Create.Step2.ReviewProposal.InputProposalName"
+      ).should("have.text", "Edited Proposal");
+
+      cy.findByTestId(
+        "Governance.Create.Step2.ReviewProposal.ConfirmDetails"
+      ).click();
+
+      cy.findByTestId("Governance.Create.Step3.SubmitProposal.Command")
+        .should("be.visible")
+        .should(
+          "have.text",
+          `creategovcfp '{"title": "Edited Proposal" ,"context":"${discussion}","amount": ${amountRequested} ,"payoutAddress":"${receivingAddress}", "cycles": 1}'`
+        );
+    });
+
+    it("should be able to cancel switching proposal when editing", () => {
+      cy.findByTestId("Governance.Create.Step1.Edit")
+        .should("be.visible")
+        .click();
+
+      cy.findByTestId("Governance.Create.Step1.RadioGroup.DFIP").click();
+      cy.findByTestId("Governance.Create.ConfirmDialog.CancelButton")
+        .should("be.visible")
+        .click();
+
+      cy.findByTestId(
+        "Governance.Create.Step1.TextArea.AmountRequested"
+      ).should("be.visible");
+      cy.findByTestId(
+        "Governance.Create.Step1.TextArea.AmountRequested"
+      ).should("be.visible");
+      cy.findByTestId("Governance.Create.Step1.TextArea.Cycles").should(
+        "be.visible"
+      );
+      cy.findByTestId(
+        "Governance.Create.Step1.TextArea.ReceivingAddress"
+      ).should("be.visible");
+    });
+
+    it("should be able to switch proposal type", () => {
+      cy.findByTestId("Governance.Create.Step1.RadioGroup.DFIP").click();
+
+      cy.findByTestId("Governance.Create.ConfirmDialog.Title").should(
+        "have.text",
+        "Confirm Edit"
+      );
+
+      cy.findByTestId("Governance.Create.ConfirmDialog.Description").should(
+        "have.text",
+        "Changing the type of proposal would cause all data to reset. Are you sure you want to continue?"
+      );
+
+      cy.findByTestId("Governance.Create.ConfirmDialog.ConfirmButton")
+        .should("be.visible")
+        .click();
+
+      cy.findByTestId("Governance.Create.Step1.RadioGroup.DFIP")
+        .should("be.visible")
+        .within(() => {
+          cy.findByTestId(
+            "Governance.Create.Step1.RadioGroup.DFIP.Checked"
+          ).should("be.visible");
+        });
+
+      cy.findByTestId("Governance.Create.Step1.TextArea.NameOfProposal").within(
+        () => {
+          cy.get("textarea").should("have.text", "");
```

Comment:
Can we please fill it up those inputs initially, and then check that they are empty

Reply:
inputs are filled from previous tests when creating CFP proposal cause this is a CFP -> DFIP switch so was checking if the text fields are empty after the switch